src/test/java/uk/co/ramp/covid/simulation/place/CareHomeTest.java

```diff
@@ -108,21 +111,17 @@ public void sickResidentsAreQuarantined() {
         inf.getcVirus().forceSymptomatic(true);
         
         Time t = new Time(0);
+        for (int i = 0; i <= inf.getcVirus().getSymptomDelay(); i++);
```

Comment:
I'm confused.  This line doesn't do anything!?!

Reply:
You're right! That's very strange - it should have replaced the while loop below (which still seems to be there; no idea what happened there!). I'll fix this up!